.circleci/config.yml

```diff
@@ -1,12 +1,18 @@
 version: 2.1
+
+orbs:
+  tools: gotest/tools@0.0.13
+
 jobs:
   lint:
     docker:
       - image: circleci/golang:1.13
     steps:
       - checkout
-      - run: make check-style
-
+      - tools/install-golangci-lint:
+          golangci-lint-version: 1.31.0
+      - tools/lint:
+          golangci-lint-version: 1.31.0
```

Comment:
I would like to use the Makefile here, as that is also what developers are using. Is this suitable?

Reply:
But that's why Orb was used, to install golangcilint.

Reply:
Can the orb be used to _only_ install golangci-lint and use `make check-style` for running it?